cmd/experimental/kjobctl/pkg/builder/templates/slurm_init_entrypoint_script.sh.tmpl

```diff
@@ -55,6 +55,9 @@ SLURM_NPROCS={{.SlurmNProcs}}
 SLURM_NNODES={{.SlurmNNodes}}
 SLURM_SUBMIT_DIR={{.SlurmSubmitDir}}
 SLURM_SUBMIT_HOST=$HOSTNAME
+SLURM_JOB_NODELIST={{.SlurmJobNodeList}}
+SLURM_NODELIST={{.SlurmJobNodeList}}
```

Comment:
Why do we have `SLURM_NODELIST` since we already have the same data in `SLURM_JOB_NODELIST`?
IIUC slurm only supports SLURM_JOB_NODELIST (https://hpcc.umd.edu/hpcc/help/slurmenv.html)  so I see no obvious need to also support another var

Reply:
It's a deprecated variable https://slurm.schedmd.com/sbatch.html#OPT_SLURM_NODELIST. But we added it as well.

Reply:
Removed.